src/components/TimeSliderSettings.vue

```diff
@@ -68,7 +68,7 @@
                 ref="endDatePicker"
                 v-model="endDate"
                 :min="startDate"
-                max="2020"
+                max="2021"
```

Comment:
not per se for this PR - but we should have one central place to set this 'year' (env or conf), now coming back in different places thoughout the code